### DISCUSSION TITLE
Support for Entity Framework?

### DISCUSSION THREAD

**Discussion by yonidagan** [quartznet/quartznet#1614] [Q&A]
Are there any plans adding support for Entity Framework or defining an IJobStore interface that will allow providing different custom stores?

**Comment by lahma** [comment]
No plain in the core project, but there seems to be community works [like this one](https://github.com/appany/AppAny.Quartz.EntityFrameworkCore.Migrations) (supports Postgres).

**Reply by tonyqus** [reply]
I'm reviewing [AppAny.Quartz.EntityFrameworkCore.Migrations](https://github.com/appany/AppAny.Quartz.EntityFrameworkCore.Migrations). The release of this component only supports .NET 6. 

I'm checking with the author if he is willing to support old .NETs.

**Comment by tonyqus** [comment]
@yonidagan Would you please let me know your scenario? I think Entity framework integration is good and important for extending features based on Quartz.NET.

**Reply by yonidagan** [reply]
I would like to use EF as the job persistence provider for Quartz.NET. That is different than only using Migrations and using SQL DB provider. Using EF could actually mean it is set up with any of the DBs EF supports.
